fastlane_core/lib/fastlane_core/feature/feature.rb

```diff
@@ -0,0 +1,11 @@
+module FastlaneCore
+  class Feature
+    attr_accessor :key, :description, :env_var, :experiment
+    def initialize(key:, description:, env_var:, experiment: true)
+      @key = key
+      @description = description
+      @env_var = env_var
+      @experiment = experiment
+    end
+  end
+end
```

Comment:
no idea why this is happening. i have a new line! 
![screen shot 2016-06-10 at 11 47 22 am](https://cloud.githubusercontent.com/assets/2587018/15970127/1d784bac-2f01-11e6-9bb7-d90f8352962a.png)